src/yang/ietf-access-control-list.yang

```diff
@@ -466,60 +466,71 @@ module ietf-access-control-list {
               "The rules in this set determine what fields will be
                matched upon before any action is taken on them.
                The rules are selected based on the feature set
-               defined by the server and the acl-type defined.";
-
-            container eth {
-              when "derived-from(../../../../acl-type, " +
-                   "'acl:eth-acl-type')";
-              if-feature match-on-eth;
-              uses packet-fields:acl-eth-header-fields;
+               defined by the server and the acl-type defined.
+               If the packet does not match any of the fields
```

Comment:
Maybe reword to:

```
Any field not specified is effectively wild carded. Thus if no
matches are specified, it is the equivalent of 'any'."
```

I'll put this in my PR.

Reply:
How about this?

If no matches are specified, it is the equivalent of 'any'.